src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ReleaseDateService.kt

```diff
@@ -25,7 +25,7 @@ class ReleaseDateService(
     val hardStopDate = getHardStopDate(sentenceDateHolder)
     val today = LocalDate.now(now)
 
-    if (hardStopDate == null) {
+    if (hardStopDate == null || sentenceDateHolder.licenceStartDate == null) {
```

Comment:
some test data failed on this - should check but imagine we probably have some licences without a start date in the DB

Reply:
We definitely will because of IS91 sentence deactivation etc